tests/test_signing.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+import tempfile
+from collections.abc import Mapping
+from pathlib import Path
+
+import pytest
+
+from mkosi.run import find_binary, run
+
+from . import Image, ImageConfig
+
+pytestmark = pytest.mark.integration
+
+
+def test_signing_checksums_with_sop(config: ImageConfig) -> None:
+    assert config.tools is not None, "Needs config.tools to check for SOP implementation"
+    if find_binary("sqop", config.tools) is None:
+        print("Needs `sqop` binary in PATH to perform sop tests.")
+        return
```

Comment:
Use `pytest.skip` please

Reply:
Okay. I assumed `pytest.skip` wouldn't be able to access the `config` parameter of the function but I didn't actually test this :roll_eyes: